bedrock/firefox/tests/test_views.py

```diff
@@ -998,7 +998,7 @@ def test_accounts_page_legacy(self, render_mock):
         req = RequestFactory().get('/firefox/accounts/')
         req.locale = 'fr'
         views.firefox_accounts(req)
-        render_mock.assert_called_once_with(req, 'firefox/accounts.html', ANY)
+        render_mock.assert_called_once_with(req, 'firefox/accounts-2019.html', ANY)
```

Comment:
I don't think this particular test (`test_accounts_page_legacy`) is needed any longer. You can remove this test, as opposed to update the template name here.